File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Commons/BuildActionModel.ts
import BuildActionType = require('App/Models/Commons/BuildActionType');
import ReferenceBuildModelType = require('App/Models/Commons/ReferenceBuildModelType');

interface BuildActionModel {
    BuildType: BuildActionType;
    Model: string;
    ReferenceId: string;
    ReferenceType: ReferenceBuildModelType;
}

export = BuildActionModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Common/Infrastructures/BuildActionModel.cs
using System;

namespace VecompSoftware.DocSuiteWeb.Common.Infrastructures
{
    public class BuildActionModel
    {
        public Guid ReferenceId { get; set; }

        public string Model { get; set; }

        public BuildActionType BuildType { get; set; }
    }
}
